lua/methods/atmosphere.lua

```diff
@@ -0,0 +1,56 @@
+--
+-- RPC atmosphere actions
+-- https://wiki.hoggitworld.com/view/DCS_singleton_atmosphere
+--
+
+local atmosphere = atmosphere
+local coord = coord
+local GRPC = GRPC
+
+local function convertWindResults(windvec3)
+  local direction = math.deg(math.atan2(windvec3.z, windvec3.x))
+  
+  if direction < 0 then
+    direction = direction + 360
+  end
+  
+  -- Convert TO direction to FROM direction. 
+  if direction > 180 then
+    direction = direction-180
+  else
+    direction = direction+180
+  end
+  
+  -- Calc 2D strength.
+  local strength = math.sqrt((windvec3.x)^2+(windvec3.z)^2)
+
+  return  {
+    heading = direction, -- Western style. Heading the wind is coming FROM
```

Comment:
I think `direction` is usually used to denote the direction the wind is coming from, while `heading` would the direction the wind is going to

Reply:
I wanted to avoid direction because that is what was used in the ME but yeah we should probably stick to it. Changed "strength" to "speed"